analyzers/tests/SonarAnalyzer.Test/LiveVariableAnalysis/RoslynLiveVariableAnalysisTest.FlowCaptureOperation.cs

```diff
@@ -0,0 +1,354 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2024 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+extern alias csharp;
+namespace SonarAnalyzer.Test.LiveVariableAnalysis;
+
+public partial class RoslynLiveVariableAnalysisTest
+{
+    [TestMethod]
+    public void FlowCaptrure_NullCoalescingAssignment()
+    {
+        /*      Block 1
+        *     #0=stringParam
+        *          |
+        *       Block 2
+        *        #1=#0
+        *      #1 is null --+
+        *          |        |
+        *       Block 3     |
+        *       #0=Hello    |
+        *          |        |
+        *        Exit <-----+
+        */
+        const string code = """stringParam ??= "Hello";""";
+        var context = CreateContextCS(code, additionalParameters: "string stringParam");
+        context.ValidateEntry(LiveIn("stringParam"), LiveOut("stringParam"));
+        context.Validate(context.Cfg.Blocks[1], null, LiveIn("stringParam"), LiveOut("stringParam"));
```

Comment:
There are so many that it makes sense to add one more overload into the context
```suggestion
        context.Validate(context.Cfg.Blocks[1], LiveIn("stringParam"), LiveOut("stringParam"));
```